release/_releases_json.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/bash
 
-function regenerate_releases_json {
-	_process_product dxp
-	_process_product portal
+function generate_releases_json {
+	if [ "${1}" = "regenerate" ]
+	then
+		_process_product dxp
+		_process_product portal
+	else
+		local status=$(_process_new_product)
```

Comment:
I removed the separated dxp and portal check due to the change from parameter to `"${LIFERAY_RELEASE_PRODUCT_NAME}"`. This way the exit code checking also got more simple.

Reply:
Good job, @Wykenz! :clap: :smile: 

I just see one more optimization. How about this?
```
if [ "$(_process_new_product)" -eq "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ]
then
		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
fi
```